templates/build/news-feed-base.build.hbs

```diff
@@ -69,6 +69,20 @@
       \{{/if}}
     </div>
     \{{/ifCond}}
+    \{{#ifCond filtersEnabled '!!' bookmarksEnabled}}
```

Comment:
1. You don't need another `ifCond` block. You should be able to use `{{else}}` to achieve the same result and not having to create a new condition. See https://stackoverflow.com/questions/11327617/handlebars-custom-if-helper-else-is-undefined/11327911
2. Can we find a way to note repeat the sort UI markup? Essentially, we currently have a condition that shows sort UI if filter or bookmark is enabled. We need to update this so that sort UI is shown if filter, bookmark or sort is enabled. This should be achievable without needing to repeat the sort UI markup.

Reply:
1. I've got this error 
![image](https://user-images.githubusercontent.com/74735207/103762424-eff63e00-5020-11eb-99ba-60beb38a3d21.png)

2. Can we try this https://stackoverflow.com/a/21915381 ?

Reply:
I don't see how https://stackoverflow.com/a/21915381 is relevant.

We currently have a condition like this:

```
{{#ifCond filtersEnabled '||' bookmarksEnabled}}
```

The condition doesn't check whether sorting is enabled.

Can we change it to 

```
{{#if searchIconsEnabled}}
```

And `searchIconsEnabled` can be a result of `filtersEnabled || bookmarksEnabled || sortEnabled`.

This way you don't need to repeat the sort UI markup or worry about any `else` clauses.

Reply:
Updated with new condition.